.github/workflows/build-push.yml

```
@@ -13,17 +13,15 @@ jobs:
     runs-on: ubuntu-latest
     permissions: 
       contents: read
+      id-token: write
       
     steps:
     - name: Login to Docker Hub
-      uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
-      with:
-        username: ${{ secrets.DOCKERHUB_USERNAME }}
-        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      uses: grafana/shared-workflows/actions/dockerhub-login@117d8511cbc5da0337972deeb400c4298b057af3 #v1.0.1
```

The tag here needs to come from https://github.com/grafana/shared-workflows/releases?q=dockerhub-login&expanded=true so it should be `# dockerhub-login-v1.0.1` instead of `#v1.0.1`

```suggestion
      uses: grafana/shared-workflows/actions/dockerhub-login@117d8511cbc5da0337972deeb400c4298b057af3 # dockerhub-login-v1.0.1
```

---

Ahhh I see, thank you so much!